src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesApiServiceTest.kt

```diff
@@ -188,22 +190,32 @@ internal class ActivitiesApiServiceTest {
       val schedule = activitiesApiService.getActivitySchedule(1234)
 
       assertThat(schedule.id).isEqualTo(1234)
+      assertThat(schedule.startDate).isEqualTo("2023-01-20")
+      assertThat(schedule.endDate).isEqualTo("2023-12-24")
       assertThat(schedule.description).isEqualTo("Monday AM Houseblock 3")
       assertThat(schedule.internalLocation?.id).isEqualTo(98877667)
       assertThat(schedule.internalLocation?.code).isEqualTo("EDU-ROOM-1")
       assertThat(schedule.capacity).isEqualTo(10)
-      val activity = schedule.activity
-      assertThat(activity.id).isEqualTo(123456)
-      assertThat(activity.prisonCode).isEqualTo("PVI")
-      assertThat(activity.description).isEqualTo("A basic maths course suitable for introduction to the subject")
-      assertThat(activity.category.code).isEqualTo("LEISURE_SOCIAL")
-      assertThat(activity.minimumIncentiveNomisCode).isEqualTo("BAS")
+      assertThat(schedule.activity.payPerSession).isEqualTo(ActivityLite.PayPerSession.F)
+      assertThat(schedule.activity.minimumIncentiveNomisCode).isEqualTo("BAS")
       with(schedule.instances.first()) {
         assertThat(date).isEqualTo("2022-12-30")
         assertThat(startTime).isEqualTo("9:00")
         assertThat(endTime).isEqualTo("10:00")
       }
-      // TODO: pay, start/end date
```

Comment:
Had a general review and added assertions for any data returned from Activities that we're using.